kolla-template-overrides.j2

```diff
@@ -41,6 +41,7 @@ RUN pip --no-cache-dir install --upgrade \
 
 # See "neutron-server-additions-extra" section in kolla-build.conf
 {% block neutron_server_footer %}
+RUN pip install netmiko==3.4.0
 ADD additions-archive /
 RUN cp -R /additions/extra/neutron-server/* /etc/neutron/ \
```

Comment:
I'm surprised these patches still apply cleanly (and do we still need these patches?)

Reply:
I'm not sure actually, they only apply to the dellforce10 driver (Dell os9). Possibly it's needed at TACC?

Reply:
the Dell FNOIA patches are needed to support the "pass-through" devices in the Dell FX2 Chassis. They aren't quite standard switches, but need management anyway.